src/main/java/com/javadiscord/javabot/properties/config/guild/HelpConfig.java

```diff
@@ -0,0 +1,79 @@
+package com.javadiscord.javabot.properties.config.guild;
+
+import com.javadiscord.javabot.help.AlphabetNamingStrategy;
+import com.javadiscord.javabot.help.AnimalNamingStrategy;
+import com.javadiscord.javabot.help.ChannelNamingStrategy;
+import com.javadiscord.javabot.properties.config.GuildConfigItem;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import net.dv8tion.jda.api.entities.Category;
+
+/**
+ * Configuration for the guild's help system.
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class HelpConfig extends GuildConfigItem {
+	/**
+	 * The id of the channel category that all help channels are in.
+	 */
+	private long categoryId;
+
+	/**
+	 * The strategy to use when naming help channels.
+	 */
+	private String channelNamingStrategy = "animal";
+
+	/**
+	 * The number of open help channels to maintain.
+	 */
+	private int preferredOpenChannelCount = 3;
+
+	/**
+	 * The string which is prefixed to any open help channel, where users are
+	 * free to ask a question.
+	 */
+	private String openChannelPrefix = "\uD83D\uDFE2";
+
+	/**
+	 * The string which is prefixed to any reserved help channel, where a user
+	 * has already asked a question and is in the process of getting an answer.
+	 */
+	private String reservedChannelPrefix = "⛔";
```

Comment:
openChannelPrefix uses escaped unicode sequences and reservedChannelPrefix uses a unicode emoji directly embedded in the code.

Reply:
It is fixed.